src/main/java/org/opensearch/jdbc/types/TimeType.java

```diff
@@ -24,41 +30,56 @@ public Time fromValue(Object value, Map<String, Object> conversionParams) throws
     if (value == null) {
       return null;
     }
+    Calendar calendar = conversionParams != null ? (Calendar) conversionParams.get("calendar") : null;
     if (value instanceof Time) {
-      return asTime((Time) value);
+      return (Time) value;
```

Comment:
Should the returned value here be converted to localtime similar to the case where `value` is a string?

Reply:
I don't see what adding that extra step will accomplish in this case, so I return `value` as `java.sql.Time` when the value is of type `java.sql.Time`. In case of string, value needs to be checked and converted correctly, so LocalTime is used to do so.

Reply:
Gocha, thanks!